helix-core/src/main/java/org/apache/helix/util/StatusUpdateUtil.java

```diff
@@ -55,6 +56,9 @@
 public class StatusUpdateUtil {
   static Logger _logger = LoggerFactory.getLogger(StatusUpdateUtil.class);
 
+  public static final boolean ERROR_LOG_TO_ZK_ENABLED =
```

Comment:
nit, private?
Since we already doing reflection, I think making it private won't block the test, right?

Reply:
Actually it is not the case. Making it private won't work as I just test. I am not an expert in this area. If you know other trick, let me know.

Reply:
This is not the case. Just tested, it does not work. I am not an expert in this area. Feel free to let me know if you know the trick here.